src/diffusers/loaders/lora.py

```diff
@@ -66,6 +67,35 @@
 LORA_DEPRECATION_MESSAGE = "You are using an old version of LoRA backend. This will be deprecated in the next releases in favor of PEFT make sure to install the latest PEFT and transformers packages in the future."
 
 
+def text_encoder_attn_modules(text_encoder):
```

Comment:
We need those for PEFT for now, so to be able to easily delete `src/diffusers/models/lora.py` let's move it here.